### PR TITLE
Update index.md

### LINKED ISSUE #33185: Logs that begin with "Microsoft" categories are from ASP.NET Core framework
### Description

More accurately, .NET

### Page URL

https://learn.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-8.0

### Content source URL

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/fundamentals/logging/index.md

### Document ID

726e3bf1-f367-d733-8933-bccc04da0e16

### Article author

@tdykstra

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #33185

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/fundamentals/logging/index.md](https://github.com/dotnet/AspNetCore.Docs/blob/d0d3b6bc54f89c56eed40f8c155a59a019ddd4a6/aspnetcore/fundamentals/logging/index.md) | [Logging in .NET Core and ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/logging/index?branch=pr-en-us-33186) |

### CODE REVIEW

File: aspnetcore/fundamentals/logging/index.md [hunk 1]
```diff
@@ -218,7 +218,7 @@ Logs created with the [default logging providers](#lp) are displayed:
   * In the ASP.NET Core Web Server window.
 * In the console window when the app is run with `dotnet run`.
 
-Logs that begin with "Microsoft" categories are from ASP.NET Core framework code. ASP.NET Core and application code use the same logging API and providers.
+Logs that begin with "Microsoft" categories are from .NET. .NET and application code use the same logging API and providers.
```
Comment (Rick-Anderson) [inline aspnetcore/fundamentals/logging/index.md:221]:
```suggestion
Logs that begin with "Microsoft" categories are from .NET. .NET and application code use the same [logging API and providers](/dotnet/core/extensions/logging?tabs=command-line).
```

File: aspnetcore/fundamentals/logging/index.md [hunk 2]
```diff
@@ -408,9 +408,9 @@ The preceding code displays console logs when the category contains `Controller`
 
 Generally, log levels should be specified in configuration and not code.
 
-## ASP.NET Core and EF Core categories
+## ASP.NET Core categories
 
-The following table contains some categories used by ASP.NET Core and Entity Framework Core, with notes about the logs:
+The following table contains some categories used by ASP.NET Core:
```
Comment (guardrex) [inline aspnetcore/fundamentals/logging/index.md:413]: Per the style manual.

**Lines 243, 538, 547, and 558 should also get the same change with some wording updates to say 'in the following table' or similar.**

```suggestion
The following table contains some categories used by ASP.NET Core.
```

File: aspnetcore/fundamentals/logging/index.md [hunk 3]
```diff
@@ -422,14 +422,16 @@ The following table contains some categories used by ASP.NET Core and Entity Fra
...
-| `Microsoft.EntityFrameworkCore`       | General Entity Framework Core diagnostics. Database activity and configuration, change detection, migrations. |
+
 
 To view more categories in the console window, set **`appsettings.Development.json`** to the following:
 
 [!code-json[](~/fundamentals/logging/index/samples/3.x/MyMain/appsettings.Trace.json)]
 
 <!-- Review: What other providers support scopes? Console is not generally used in staging/production  -->
 
+See [EF Message categories](/ef/core/logging-events-diagnostics/simple-logging#message-categories) for a list of Entity Framework categories.
```
Comment (guardrex) [inline aspnetcore/fundamentals/logging/index.md:433]: Optional ... I usually start with the prep phrase ...

```suggestion
For a list of Entity Framework categories, see [EF Message categories](/ef/core/logging-events-diagnostics/simple-logging#message-categories).
```

**Review by guardrex — APPROVED** [review]
One NIT 😈 ... Sentences preceding tables should end with a period.

Cross-ref: https://learn.microsoft.com/en-us/style-guide/scannable-content/tables#punctuation

**Pull request merged**